sync/Dockerfile

```diff
@@ -12,12 +12,12 @@ ARG PIP_NO_CACHE_DIR="off" \
     PYTHONUNBUFFERED="1"
 
 # Copy files and install requirements
-WORKDIR /app
+WORKDIR /sync
```

Comment:
Please do not change the default Docker directory.  Leave it as `/app`.

Reply:
I will undo that change